client/my-sites/plugins/marketplace/marketplace-plugin-details/purchase-area.tsx

```diff
@@ -18,6 +18,7 @@ interface PurchaseArea {
 	onNavigateToCheckout: () => void;
 	onNavigateToDomainsSelection: () => void;
 	onRemoveEverythingFromCart: () => Promise< ResponseCart >;
+	onInstallPluginManually: ( { primaryDomain }: { primaryDomain: string } ) => Promise< void >;
```

Comment:
It appears to me that it can just accept a string, i.e. `( string ) => Promise< void >`. Is there a reason that the parameter has to be an object?

Reply:
Fixed by 09f91fa3b7e58c4dbbbd741a54e95d686bf21d82